src/translation/translation.h

```diff
@@ -97,6 +97,8 @@ typedef enum {
     TR_HOTKEY_SET_BOOKMARK_4,
     TR_HOTKEY_EDITOR_TOGGLE_BATTLE_INFO,
     TR_HOTKEY_EDIT_TITLE,
+    TR_HOTKEY_DUPLICATE_TITLE,
+    TR_HOTKEY_DUPLICATE,
```

Comment:
Rename to `TR_HOTKEY_DUPLICATE_MESSAGE` to indicate it's the message for the dialog.